frontend/components/Site/Nav/Nav.tsx

```diff
@@ -1,63 +1,77 @@
 import { width, darkGrey } from '../../../utils'
 import NavLink from '../../NavLink'
 import Logo from '../../Logo'
+import { useCurrentUserQuery, User as UserType } from '../../../generated/graphql'
 
-const navItems = [
-  { name: 'About', path: '/about' },
-  { name: 'Blog', path: '/blog/introducing-journaly' },
-  { name: 'Log in', path: '/dashboard/login' },
-  { name: 'Sign up', path: '/dashboard/signup' },
-]
-
-const Nav = () => (
-  <div>
-    <div className="header-container">
-      <Logo />
-
-      <ul className="nav-items">
-        {navItems.map((navItem) => (
-          <NavLink href={navItem.path} key={navItem.name}>
-            <a className="nav-link">{navItem.name}</a>
+const Nav = () => {
+  const { data } = useCurrentUserQuery()
+  const currentUser = data?.currentUser as UserType
+
+  return (
+    <div>
+      <div className="header-container">
+        <Logo />
+
+        <ul className="nav-items">
+          <NavLink href="/about" key="About">
```

Comment:
We can remove all the `key`s if we aren't looping over with map anymore

Reply:
Good point!